etc/apache-mod_proxy_fcgi.conf.template

```diff
@@ -0,0 +1,77 @@
+###
```

Comment:
As a number of templates started growing and perhaps we will add more and more, does it make sense to put them in etc/templates or similar?